state.json

```
@@ -242,7 +242,8 @@
                 "name": "pocket",
                 "random": true,
                 "commands": [
-                    "https://i.imgur.com/EgoNBWl.jpg comment=\"Wifi update success\" end=2018-11-24T00:00:00"
+                    "https://i.imgur.com/EgoNBWl.jpg comment=\"Wifi update success\" end=2018-11-24T00:00:00".
```

This needs to be a comma instead of a period.

```suggestion
                    "https://i.imgur.com/EgoNBWl.jpg comment=\"Wifi update success\" end=2018-11-24T00:00:00",
```